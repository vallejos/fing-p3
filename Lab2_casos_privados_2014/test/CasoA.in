7
1 2 -1
0 -1
3 -1
4 -1
-1
6 -1
5 -1
Fauna 5 Flora 1 -1 -1
Fauna 6 Flora 8 -1 -1
Fauna 7 Flora 3 -1 -1
Campo 6 Fauna 8 -1 -1
Campo 4 Agua 11 -1 -1
Agua 10 -1 -1
Agua 3 -1 -1
identificarColecciones 4 Fauna Flora Agua Campo
salir
