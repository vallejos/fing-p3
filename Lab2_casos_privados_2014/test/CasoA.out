Tarea 2 - 2014 - Programacion 3.

>0 -> Flora
1 -> Fauna
2 -> Campo
3 -> 
4 -> Agua
>FIN.
